etp-front/src/language/fi.json

```diff
@@ -388,6 +388,9 @@
         "select-yritys": "Valitse liitettävä yritys",
         "attach-failed": "Liittäminen yritykseen epäonnistui",
         "detach-failed": "Laatijan poistaminen yrityksestä epäonnistui"
+      },
+      "screen-reader": {
+        "search-results": "{count} hakutulosta pudotusvalikossa. Valitse yritys listasta."
       }
```

Comment:
Tulokset tulevat pudotusvalikkoon, joka on lista. Nyt käyttäjä voi varmaan osata käyttää ruudunlukijan listanavigointia. Lista kuitenkin katoaa, jos pudotusvalikko ei ole esillä, joten laitoin tuon pudotusvalikon mainitsemisenkin tuohon.

Reply:
Luulen että jos tämmöisen lisäohjeen haluaa tarjota, niin se olisi parempi laittaa erillisenä staattisena elementtinä esim sr-only -classilla varustettuna. Näin sitä ei turhaan luettaisi joka kerta, mutta sen voi kuitenkin selata kuuluville.